### DISCUSSION TITLE
Can we standardize a Copyright & TOS component?

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2487] [Ideas] [answered]
I found these in <5 mins 🙈
https://www.figma.com/file/bPtdTbbouKF3kCLFRFJHOG/ToS-pileup?node-id=0%3A1

**Comment by serifluous** [comment, marked as answer]
Hey @rosemaryluo ! Thanks for sharing. We've actually been capturing this use case as part of the "small/detail/caption text" item you can find on our roadmap: https://paste.twilio.design/roadmap#2022-11-29-candidate

If you have any initial thoughts on how the copyright + ToS text should be formatted, please share! Something that would really help us, too, is if you could share screenshots of the full pages where you grabbed those examples from.

**Reply by rosemaryluo** [reply]
I don't know anything about the requirements around Copyright / ToS at the moment or who would have that information, my concern is only consistency since a lot of these pages can be seen in quick succession during onboarding and it's one of those I don't expect attention on but influences the overall sense of safety / trust with Twilio.

Happy to grab screenshots--do you have a rec for a full page screenshot tool?

**Reply by serifluous** [reply]
Nice, super helpful.

I use this extension: https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl

And you can drop the screenshots here: https://www.figma.com/file/6MlUZI1JHp8aIWE5JCGlm0/Paste-Moodboard?node-id=1407%3A66639

**Reply by rosemaryluo** [reply]
Hi @serifluous , finally got around to adding these! Sorry for the delay

**Reply by rosemaryluo** [reply]
Is there an update on when Paste would have this out? I no longer see it on the roadmap.

**Reply by serifluous** [reply]
Hey @rosemaryluo, I chatted with @bhamtil-twilio, and we agreed that this might be better served as a Console pattern. I'll let y'all coordinate that. I think the screenshots you dropped into the Figma moodboard is still a pretty good foundation for the work.
